oak_remote_attestation_noninteractive/proto/v1/BUILD

```diff
@@ -34,6 +34,12 @@ proto_library(
     deps = [":messages_proto"],
 )
 
+proto_library(
+    name = "service_unary_proto",
+    srcs = ["service_unary.proto"],
```

Comment:
Is `service_unary.proto` already in the repo?

Reply:
I don't believe so. There is only service_streaming.proto for noninteractive remote attestation.

Reply:
Looks like it didn't get picked up in my commit. Adding now